bench-tps/src/bench.rs

```diff
@@ -25,8 +25,9 @@ use std::thread::Builder;
 use std::time::Duration;
 use std::time::Instant;
 
-pub const MAX_SPENDS_PER_TX: usize = 4;
-pub const NUM_LAMPORTS_PER_ACCOUNT: u64 = 20;
+pub const MAX_SPENDS_PER_TX: u64 = 4;
+pub const NUM_LAMPORTS_PER_ACCOUNT: u64 = 128;
+pub const NUM_LAMPORTS_PER_TX: u64 = 1;
```

Comment:
Why not fetch this from the cluster when we get a blockhash

Reply:
sounds good. I was wondering how to get this value during runtime..

Reply:
Done